pandas/tests/io/test_excel.py

```diff
@@ -342,15 +342,15 @@ def test_excel_passes_na(self, ext):
         tm.assert_frame_equal(parsed, expected)
 
     @td.skip_if_no('xlrd', '1.0.1')  # GH-22682
-    def test_deprecated_sheetname(self, ext):
+    @pytest.mark.parametrize('arg', ['sheet', 'sheetname'])
+    def test_deprecated_sheetname(self, ext, arg):
```

Comment:
Let's make this test name more generalizable (e.g. `test_unexpected_kwargs_raises`) to keywords that we explicitly `TypeError` for (since they're no longer deprecated, just removed).

Reply:
Speaking of that...it looks `parse_cols` is going to be dropped as well, which only serves to strengthen my assertion that this test name should be generalized.

Reply:
I'll address `parse_cols` in another PR.